app/models/search_parser.rb

```diff
@@ -4,6 +4,8 @@
 
 # https://mariadb.com/kb/en/full-text-index-stopwords/#innodb-stopwords
 MYISAM_STOPWORDS = %w[a about an are as at be by com de en for from how i in is it la of on or that the this to was what when where who will with und the www].sort_by { it.length }.reverse.freeze
+# curl -s https://data.iana.org/TLD/tlds-alpha-by-domain.txt | grep -v "^#" | tr '\n' ' ' | tr '[:upper:]' '[:lower:]'
```

Comment:
We've had a few constants like this over the years and never manged to keep them up-to-date before we started writing jobs to do it.

Take a look at `config/recurring.yml` and use `fetch_email_blocklist` as a template for a monthly `FetchTLDsJob`. Then instead of the strategy in `app/models/concerns/email_blocklist_validation.rb` of falling through to `false` when the file doesn't exist, return a fallback list of %w[com org net ping co]` with a comment that it's for the SearchParser specs.